src/iso22133object.cpp

```diff
@@ -170,62 +145,43 @@ void TestObject::sendGREM(HeaderType msgHeader, GeneralResponseStatus responseCo
 }
 
 void TestObject::sendMonrLoop() {
-	std::stringstream ss;
-	ss << "Started MONR thread." << std::endl;
-	std::cout << ss.str();
-
-	while (this->on && ctrlChannel.isOpen()) {
-		// Only send monr if transmitterID has been set by an OSEM message
-		if (this->transmitterID != TRANSMITTER_ID_UNAVAILABLE_VALUE){
-			sendMONR();
+	while (this->on) {
```

Comment:
Investigate possibilities of removing the need for busy waiting by implementing a boost queue, to only do work when necessary

Reply:
https://stackoverflow.com/questions/4267546/c-boost-asio-simple-periodic-timer
The first answer is an example of functionality equivalent to WallTimer in ros2